src/hotspot/share/cds/heapShared.cpp

```diff
@@ -544,7 +540,7 @@ void HeapShared::archive_java_mirrors() {
     oop m = scratch_java_mirror(orig_k);
     if (m != nullptr) {
       copy_java_mirror_hashcode(orig_mirror, m);
-      copy_aot_initialized_mirror(orig_k, orig_mirror, m);
+//    copy_aot_initialized_mirror(orig_k, orig_mirror, m);
```

Comment:
Should this be deleted rather than commented?

Reply:
It has been removed from the latest patch.